src/weekday.rs

```diff
@@ -50,14 +50,29 @@ pub enum Weekday {
 }
 
 impl Weekday {
+    /// Create a `Weekday` from an `u32`, with Monday = 0.
+    /// Infallible, takes `n % 7`.
+    #[inline]
+    pub(crate) const fn from_u32_mod7(n: u32) -> Weekday {
```

Comment:
It's a trivial function but for due diligence, can you add test cases for this in the `mod test` section below?

e.g.
```rust
    #[test]
    fn test_from_u32_mod7() {
        //...
    }
```